README.md

```diff
@@ -0,0 +1,137 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3> 🗺️ <b>Countries Metric App</b> 🗺️</h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [:camera: project screenshot](#screen-shoot)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 🗺️ Countries Metrics App<a name="about-project"></a>
+
+ <p>A Mobile web application offers information about european countries such as their population, area, flag, and other relevant details. Users can access a wide range of country-specific information, making it an invaluable resource for personal or professional research purposes.
+</p>
+
+## 📷 project screenshot <a name="screen-shoot"> </a>
+
+<img src="https://user-images.githubusercontent.com/40009354/225965283-c30c4010-82db-4425-bbeb-28da428471d4.PNG" alt="logo" width="auto"  height="auto" />
+
+## 🛠 Built With <a name="built-with"> </a>
+
+> React, Redux , MUI react and external API
+
+### Tech Stack <a name="tech-stack"></a>
+
+> React, Redux, Axios, Jest, Webpack and Babel
+
+### Key Features <a name="key-features"></a>
+
+ <li>Display total Europe population</li>
+<li>Display Countries with their area, population, etc.</li>
+<li>Detail information about selected country</li>
+<li>Mobile web Responisive layout</li>
+<li>Used REST countries API <a href="https://restcountries.com/">Link</a></li>
+<p align="right">(<a href="#readme-top">back to top</a>)</p><!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://country-metrics-app.onrender.com/">Live Link</a> <br>
+  -Video Presentation Link <a href="https://www.loom.com/share/3f8acfbc1d5145e48b77f004d3a67967">Live Link</a>
```

Comment:
- [ ]  Your presentation videos should  contains a highlight about a piece of the code that you find interesting or you proud of making as well. It will make your videos presentation to diplay the technical  part as of your project and  that you've enjoyed to build your project.